notebooks/samples/pytorch/lightning/TrainingAndPredictionWithPyTorchLightning.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>nit: it would be better if we copied the sonar data to the ai-platform directory within the cloud-samples-data bucket, and also updated this notebook accordingly, just to get rid of this ml-engine reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>